frontend/src/components/common/BackButton.tsx

```diff
@@ -4,16 +4,21 @@ import { Button } from "@chakra-ui/react";
 
 import { ArrowBackOutlineIcon } from "../../assets/icons";
 
-const BackButton = (): React.ReactElement => {
+interface BackButtonProps {
+  size?: string;
+  text?: string;
+}
+
+const BackButton = ({ size, text }: BackButtonProps): React.ReactElement => {
```

Comment:
maybe cleaner if we put here?
```suggestion
const BackButton = ({ size = "sm", text = "Back" }: BackButtonProps): React.ReactElement => {
```

Reply:
sorry im acc brain dead today